README.md

```diff
@@ -60,26 +120,8 @@ so that anyone had the opportunity of commenting on them, and make up their mind
 * We use the [CHAOSS mailing list](https://chaoss.community/participate/#user-content-join-the-mailing-list).
 Please prefix the message subject with \[wg-gmd\] if it is specific to this working group.
 
-## Metrics  
-
-The metrics dealt with in this focus group are organized in focus areas:
-
-* Growth Maturity and Decline
-  * [Code Development](./focus_areas/code_development.md)
-  * [Community Growth](./focus_areas/community_growth.md)
-  * [Issue Resolution](./focus_areas/issue_resolution.md)
-
-## Usage
-
-This is a repository for the GMD working group.
-When this working group decides to "release" a version of its metrics,
-the relevant files will be dumped to the
-[chaoss/metrics](https://github.com/chaoss/metrics) repository,
-as a pull request for updating the CHAOSS metrics with
-"GMD metrics release x.y".
-
 
-## Maintainers
+## Contributors
```

Comment:
Not sure if `Maintainers` should be replaced. What do you think @jgbarah ?

Reply:
Actually `Contributors` was mentioned first in the strucure proposed, so did accordingly. But  `Maintainers` is also mentioned there. Sorry :sweat_smile: I also feel that `Maintainers` would be more appropriate. 
Thanks! Let me know @valeriocos.
@jgbarah ^ Please see!

Reply:
We have both: maintainers are those with write permission in the repo, contributors are (if I'm not wrong) any other contributors.

Reply:
So what heading should appear in the README, can it both, something like:
 `Contributors / Maintainers`

Reply:
@valeriocos, @jgbarah ping!

Reply:
@jgbarah, please have a look at this. What sould come then 

> We have both: maintainers are those with write permission in the repo, contributors are (if I'm not wrong) any other contributors.
@jgbarah, just a gentle reminder, please have a look as you get time. Let me know what would be appropriate in the README.md then. Thanks!

Reply:
@harshalmittal4 you can take a look at the D&I WG. The Contributors section has a Maintainers subsection, a Core Contributor subsection and includes a list of all contributors. Maybe that is a good solution? 

https://github.com/chaoss/wg-diversity-inclusion

Reply:
Yes this seems to be a good solution, just one thing, can you please tell what all names should I include in the `Core Contributors` section. Or as of now shall I leave the `Core Contributors` header vacant?

Reply:
I don't know for GMD who the core contributors are. Maybe @sgoggins or @jgbarah know. If you don't get names, remove the section for now.

Reply:
> can you please tell what all names should I include in the `Core Contributors` section. Or as of now shall I leave the `Core Contributors` header vacant?

AFAIK, we don't have a difference except for maintainers and contributors. If the name of maintainers and/or contributors get high enough, we could discuss who is a core contributor (and what does it mean), but for now, I would remove that one.